src/dotty/tools/backend/jvm/DottyBackendInterface.scala

```diff
@@ -944,7 +944,7 @@ class DottyBackendInterface(outputDirectory: AbstractFile, val superCallsMap: Ma
   }
 
   object ValDef extends ValDefDeconstructor {
-    def _1: Modifiers = field.mods
+    def _1: Modifiers = tpd.Modifiers(field.symbol)
```

Comment:
Do you also plan to retain original positions of modifiers as written in the source file?